web-ide/proxy/Dockerfile

```diff
@@ -21,6 +21,6 @@ RUN apk update \
     && npm install \
     && npm run compile
 
-EXPOSE 80 443 3001
+EXPOSE 80 443 3001 3002
```

Comment:
Could we add comments as to what runs on each port? 80 and 443 I can guess. The others are less clear.

Reply:
👍